src/cursor/cur_backup.c

```diff
@@ -348,6 +348,10 @@ __wt_curbackup_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *other,
     if (othercb != NULL)
         WT_CURSOR_BACKUP_CHECK_STOP(othercb);
 
+    if (cfg != NULL && cfg[1] != NULL &&
+      WT_VERBOSE_LEVEL_ISSET(session, WT_VERB_BACKUP, WT_VERBOSE_LEVEL_DEFAULT))
```

Comment:
I don't see what the `WT_VERBOSE_LEVEL_ISSET` check gets us - `__wt_verbose` should already check the level very early on. But it's not incorrect.

Reply:
nice catch, i've removed it.

cc: @sueloverso